c++/include/orc/Type.hh

```diff
@@ -59,6 +74,8 @@ namespace orc {
     virtual uint64_t getMaximumLength() const = 0;
     virtual uint64_t getPrecision() const = 0;
     virtual uint64_t getScale() const = 0;
+    virtual const std::string& getCRS() const = 0;
+    virtual geospatial::EIAlgo getEIAlgo() const = 0;
```

Comment:
```suggestion
    virtual const std::string& getCrs() const = 0;
    virtual geospatial::EdgeInterpolationAlgorithm getAlgorithm() const = 0;
```

It would be good to add comment to say these two functions are for geometry & geography types only.